rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

```diff
@@ -56,6 +56,24 @@ You cannot change the machine pool node type or size. The machine pool node type
 ====
 * You can add a label to each added machine pool.
 
+*Optional:* Add a label to the default machine pool after configuration by using the default machine pool labels and running the following command:
```

Comment:
```suggestion
.Procedure
* Add a label to the default machine pool after configuration by using the default machine pool labels and running the following command:
```

I get that this isn't a required thing, BUT it's the only step of this procedure.